browser/net/brave_static_redirect_network_delegate_helper_unittest.cc

```diff
@@ -75,7 +75,7 @@ TEST(BraveStaticRedirectNetworkDelegateHelperTest, ModifyCRLSet3) {
       "https://www.google.com/dl/release2/chrome_component/LLjIBPPmveI_4988/"
       "4988_all_crl-set-6296993568184466307.data.crx3");
   const GURL expected_url(
-      "https://crlsets.brave.com/dl/release2/chrome_component/LLjIBPPmveI_4988/"
+      "https://redirector.brave.com/dl/release2/chrome_component/LLjIBPPmveI_4988/"
```

Comment:
some of these longs lines should be clang-formatted

Reply:
thank you, have updated